changelogs/fragments/72197-upgrade-test-ansible-runner.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
```

Comment:
I'm not sure this is a minor change per [our definition](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). Test changes are a somewhat gray area, which we usually file under `bugfixes` so we can backport them.

@sivel What do you think (since Matt Clay is out)?

Reply:
We have definitions!?  Honestly I use `minor_changes` that don't match that description all the time.  It's a minor change, that isn't actually related to a bug fix.  I'm not sure it's worth worrying about.

Reply:
For future reference, I wouldn't include a changelog fragment for changes that only touch integration tests. They're not shipped with the product, so a changelog entry seems unnecessary.